trackbasket_be/resources/shopping_list.py

```diff
@@ -34,16 +34,19 @@ def get(self,id):
     if at_risk_user is None:
       return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "At risk user not found"} } }, 400
     else:
-      shopping_list = at_risk_user.shopping_lists[-1]
-      store = at_risk_user.store[0]
-      if store is None:
-        return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "No store associated with this user"} } }, 400
+      if at_risk_user.shopping_lists == []:
+        return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "No shopping list associated with this user"} } }, 400
```

Comment:
adds conditional to address the case where an `at_risk_user` does not have a shopping list